src/pytest_check/plugin.py

```diff
@@ -57,8 +67,9 @@ def pytest_runtest_makereport(item, call):
                     e_str = str(e)
                     e_str = e_str.split('FAILURE: ')[1]  # Remove redundant "Failure: "
                     reprcrash = ReprFileLocation(item.nodeid, 0, e_str)
-                    reprtraceback = ExceptionRepr(reprcrash, excinfo)
-                    chain_repr = ExceptionChainRepr([(reprtraceback, reprcrash, str(e))])
+                    # FIXME - the next two lines have broken types
+                    reprtraceback = ExceptionRepr(reprcrash, excinfo)  # type: ignore
+                    chain_repr = ExceptionChainRepr([(reprtraceback, reprcrash, str(e))])  # type: ignore
```

Comment:
Removing the `# type: ignore`s here raises mypy errors...
```
src/pytest_check/plugin.py:71: error: Argument 1 to "ExceptionRepr" has incompatible type "ReprFileLocation"; expected "ReprTraceback"  [arg-type]
                        reprtraceback = ExceptionRepr(reprcrash, excinfo)
                                                      ^~~~~~~~~
src/pytest_check/plugin.py:71: error: Argument 2 to "ExceptionRepr" has incompatible type "ExceptionInfo[BaseException]"; expected "ReprFileLocation | None"  [arg-type]
                        reprtraceback = ExceptionRepr(reprcrash, excinfo)
                                                                 ^~~~~~~
src/pytest_check/plugin.py:72: error: List item 0 has incompatible type "tuple[ExceptionRepr, ReprFileLocation, str]"; expected "tuple[ReprTraceback, ReprFileLocation | None, str | None]"  [list-item]
                        chain_repr = ExceptionChainRepr([(reprtraceback, reprcrash, str(e))])
                                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
(Not sure if the classes changed from when this library was first written.)